frontend/public/locales/en/translation.json

```
@@ -356,12 +356,13 @@
     "clusterPool.creation.success.message": "{{name}} was successfully created.",
     "clusterPool.creation.success.title": "Cluster pool created",
     "clusterPool.destroy": "Destroy cluster pool",
-    "clusterPool.modal.scale.input": "Set desired cluster pool size",
-    "clusterPool.modal.scale.message": "Specify the desired size of <bold>{{clusterPoolName}}</bold>. Adjusting the size of the cluster pool will result in the creation or destruction of clusters; only unclaimed clusters will be destroyed if downsizing.",
+    "clusterPool.modal.scale.message": "Specify the desired size of <bold>{{clusterPoolName}}</bold>. Adjusting the size or running count of the cluster pool will result in the creation or destruction of clusters; only unclaimed clusters will be destroyed if downsizing.",
     "clusterPool.modal.scale.poolName": "Cluster pool name",
     "clusterPool.modal.scale.poolNamespace": "Cluster pool namespace",
+    "clusterPool.modal.scale.runningCount.input": "Set desired number of running clusters for this cluster pool.",
```

```suggestion
    "clusterPool.modal.scale.runningCount.input": "Set desired number of running clusters",
```

Remove the period, and "for this cluster pool" seems unnecessary.